test/scenario/multi-backend/README.rst

```diff
@@ -0,0 +1,58 @@
+=======================
+Swift Multi-backend Lab
+=======================
+
+This is a scenario which leverage fabfiles to bootstrap a SAIO (Swift All In One) cluster, as well as a Scality RING. Further, it allows automated setup of a storage policy that targets the Scality RING, as well validation of configured storage policies.
+
+The SAIO installation and the RING are intended to be installed on two different machines on the same network.
+
+Getting started
```

Comment:
Before the `Getting started` section I'd add a `Requirements` section like this

Requirements
--------------------
* Two servers (or VMs) running Ubuntu 14.04, hereafter named `<RING_HOST>` and `<SAIO_HOST>`.